diff --git a/node_modules/react-native-mlkit-ocr/android/build.gradle b/node_modules/react-native-mlkit-ocr/android/build.gradle
index c2bcb60..2af50cd 100644
--- a/node_modules/react-native-mlkit-ocr/android/build.gradle
+++ b/node_modules/react-native-mlkit-ocr/android/build.gradle
@@ -4,7 +4,7 @@ buildscript {
 
   repositories {
     google()
-    jcenter()
+    mavenCentral()
   }
 
   dependencies {
@@ -52,7 +52,6 @@ android {
 
 repositories {
   mavenCentral()
-  jcenter()
   google()
 
   def found = false
